14-include.md

```diff
@@ -0,0 +1,89 @@
+## Dependency on other compose projects
+
+A Compose project can declare dependency on another compose project, managed by another team
+or shared with others. This allows to keep a compose file reasonably complicated for the limited
+amount of resources a single team has to declare for it's own sub-domain within a larger deployment.
+
+`include` top-level section is used to define dependency to another compose application (or subdomain).
+Each path listed in this section MUST be loaded as an individual compose model with it's own project
```

Comment:
Regarding this [PR](https://github.com/compose-spec/compose-spec/pull/370), we should already apply the same format.
```suggestion
Each path listed in this section must be loaded as an individual compose model with it's own project
```